mantidimaging/core/operations/circular_mask/circular_mask.py

```diff
@@ -16,7 +16,7 @@ class CircularMaskFilter(BaseFilter):
 
     Intended to be used on: Reconstructed slices
 
-    When: To remove reconstruction artifacts near the outside of the image.
+    When: To remove reconstruction artifacts on the outside of the image.
```

Comment:
Do you mean the edge or border instead of  "on the outside"?

Reply:
Changed to "on the outer edge of the image." to be clearer.